tests/schbench

```diff
@@ -4,13 +4,12 @@
 # - runtime
 # - sleeptime_us
 # - cputime_us
+# - iterations
 
 . $LKP_SRC/lib/reproduce-log.sh
 
 cd $BENCHMARK_ROOT/schbench || die "no $BENCHMARK_ROOT/schbench"
 
-: "${worker_threads:=16}"
-: "${runtime:=300}"
 : "${sleeptime_us:=30000}"
```

Comment:
thanks, if a parameter is required (and will not have default value), prefer to have code like
	[[ -n "$group" ]] || die "Parameter \"group\" is empty"

Reply:
@SSapkal may be we can have something like 
```
[ -n "$runtime" ] || runtime=300
[ -n "$worker_threads" ] || worker_threads=16
```

Reply:
yes, default value is also ok.

Reply:
The changes are done according to the review comments.